labour_project/settings.py

```diff
@@ -252,7 +252,7 @@
 ]
 
 API_KEY_AUTH_ALLOWED_PATHS = [
-    r'^/uprn/\d+\.json',
+    r'^/uprn/\d+(\.json)?$',
```

Comment:
IF you had time, could rebase to have these fixes within the previous places, but no biggie.